universum/lib/utils.py

```diff
@@ -131,7 +147,7 @@ def function_to_run(*args, **kwargs):
     return decorated_function
 
 
-def trim_and_convert_to_unicode(line):
+def trim_and_convert_to_unicode(line: Union[bytes, str, RunningCommand]) -> str:
```

Comment:
RunningCommand? That is surprising.
Could you, please, convert it to string on the client side?

Reply:
sure

Reply:
is 'bytes' expected?

Reply:
I think monkeytype just got confused by cmd.process.ran in 'launcher'.
I added some annotations there to make things clear.

Reply:
But they didn't help to resolve the issue?

Reply:
mypy doesn't see any issues now, so I guess it's resolved

Reply:
Please ignore the last comment. I didn't see the change.